### PR TITLE
Statemachine cleanup

### PR DESCRIPTION
**Pull request by Neviothr**
**When merged this pull request will:**
- do some clean up in statemachine component.

### CODE REVIEW

File: addons/statemachine/fnc_clockwork.sqf [hunk 1]
```diff
@@ -18,7 +18,7 @@ SCRIPT(clockwork);
 
 {
     #ifdef STATEMACHINE_PERFORMANCE_COUNTERS
-    private _perfStartTime = diag_tickTime;
+        private _perfStartTime = diag_tickTime;
```
Comment (Killswitch00) [inline addons/statemachine/fnc_clockwork.sqf]: `_perfStartTime` belongs to the same scope as `_stateMachine` and `_list` below and should be indented the same amount. Conditional preprocessing should not change the indentation level.

File: addons/statemachine/fnc_clockwork.sqf [hunk 2]
```diff
@@ -98,8 +98,8 @@ SCRIPT(clockwork);
     };
 
     #ifdef STATEMACHINE_PERFORMANCE_COUNTERS
-    private _perfRunTime = diag_tickTime - _perfStartTime;
-    (GVAR(performanceCounters) select _id) pushBack _perfRunTime;
+        private _perfRunTime = diag_tickTime - _perfStartTime;
```
Comment (Killswitch00) [inline addons/statemachine/fnc_clockwork.sqf]: Same here and in all other places where this is done.

File: addons/statemachine/fnc_clockwork.sqf [hunk 3]
```diff
@@ -103,4 +103,4 @@ SCRIPT(clockwork);
     #endif
 
     false
-} count GVAR(stateMachines);
+} count GVAR(stateMachines);
```
Comment (commy2) [inline addons/statemachine/fnc_clockwork.sqf]: newline eof

**Comment by Neviothr** [comment]
@commy2 - git claims no newline :confused:

**Comment by jonpas** [comment]
Uhm no, commy added it, red line was removed, green one added. GitHub won't show an empty newline as an actual empty new line, but if there isn't one there will be a red marker.

**Comment by Neviothr** [comment]
I'm :turtle:

**Comment by commy2** [comment]
Sorry. I complained about it in a comment and then just fixed it myself afterwards.

**Pull request merged**